debezium-core/src/main/java/io/debezium/relational/Tables.java

```diff
@@ -101,7 +101,43 @@ public boolean matches(String catalogName, String schemaName, String tableName,
             };
         }
     }
+    
+    /**
+     * A mapper used to map some table columns into a Primary Key.
+     */
+    @FunctionalInterface
+    public static interface ColumnsToPkMapper {
+
+        /**
+         * Determine whether the named column should be part of the PK.
+         *
+         * @param catalogName the name of the database catalog that contains the table; may be null if the JDBC driver does not
+         *            show a schema for this table
+         * @param schemaName the name of the database schema that contains the table; may be null if the JDBC driver does not
+         *            show a schema for this table
+         * @param tableName the name of the table
+         * @param columnName the name of the column
+         * @return {@code true} if the column should be included, or {@code false} if the column should be excluded
+         */
+        boolean matches(String catalogName, String schemaName, String tableName, String columnName);
```

Comment:
Same concern as above; instead of returning a `boolean` I think this should rather return a list of PK column names.

Reply:
done